packages/SidePanel/src/SidePanel.js

```diff
@@ -18,6 +18,9 @@ const propTypes = {
   /** The content for the SidePanel. */
   children: PropTypes.node.isRequired,
 
+  /** Function that provides the container DOM element to be pushed. */
+  getPushContentRef: PropTypes.func,
```

Comment:
on the popover we are using the following convention when we want to receive an html node object, technically is not a react ref.

  /** Function that provides DOM element to use as target for positioning the popover. */
  getPositioningElement: PropTypes.func,

  /** Function that provides the scrolling DOM element that contains the popover. */
  getScrollContainer: PropTypes.func,

let's adjust the prop name?

Reply:
Sure `getPushContentContainer`?

Reply:
`getPushContentElement`? I think would be more obvious for the consumer?